examples/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+project(mococrw-examples)
+
+if(BUILD_EXAMPLES)
+    message(STATUS "Compiling examples")
+    set(EXAMPLE_SRC_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+    FIND_PACKAGE(Boost COMPONENTS program_options REQUIRED)
+
+    set(COMMON_EXAMPLE_HEADERS
```

Comment:
Is this variable used at all? If not, I'd just remove this and `$EXAMPLE_SRC_DIR` completely to clean up the file a bit.